### PR TITLE
VAULT-12112: add openapi responses for /sys/internal endpoints

### PR DESCRIPTION
**Pull request by dhuckins**
|Path|Operation|response|notes|
|----|----|----|----|
|/sys/internal/ui/feature-flags|`READ`|[link](https://github.com/hashicorp/vault/blob/b8de2c2b4e574c5fabde01c9be8c93c4e390c897/http/sys_feature_flags.go#L11-L13)| added test|
|/sys/internal/ui/mounts|`READ`|[link](https://github.com/hashicorp/vault/blob/2267b042dba5ebe278867662ea059c4bda17356d/vault/logical_system.go#L4160-L4161)||
|/sys/internal/ui/mounts/{path}|`READ`|[link](https://github.com/hashicorp/vault/blob/2267b042dba5ebe278867662ea059c4bda17356d/vault/logical_system.go#L920-L979)||
|/sys/internal/ui/namespaces|`READ`|[link](https://github.com/hashicorp/vault/blob/2267b042dba5ebe278867662ea059c4bda17356d/vault/logical_system_helpers.go#L56)|no test exists|
|/sys/internal/ui/resultant-acl|`READ`|[link](https://github.com/hashicorp/vault/blob/2267b042dba5ebe278867662ea059c4bda17356d/vault/logical_system.go#L4381-L4490)| test needs to wait until schema for `/v1/auth/token/create` is merged|
|/sys/internal/counters/requests|`READ`|[link](https://github.com/hashicorp/vault/blob/2267b042dba5ebe278867662ea059c4bda17356d/vault/logical_system.go#L4326-L4329)|returns nothing, should be deprecated|
|/sys/internal/counters/tokens|`READ`|[link](https://github.com/hashicorp/vault/blob/2267b042dba5ebe278867662ea059c4bda17356d/vault/logical_system.go#L4338-L4342)||
|/sys/internal/counters/entities|`READ`|[link](https://github.com/hashicorp/vault/blob/2267b042dba5ebe278867662ea059c4bda17356d/vault/logical_system.go#L4347-L4360)||

tests added

### CODE REVIEW

File: vault/logical_system_paths.go [hunk 1]
```diff
@@ -1044,6 +1162,8 @@ func (b *SystemBackend) internalPaths() []*framework.Path {
 				logical.ReadOperation: &framework.PathOperation{
 					Callback: b.pathInternalCountersRequests,
 					Summary:  "Backwards compatibility is not guaranteed for this API",
+					// callback only returns errors
```
Comment (dhuckins) [inline vault/logical_system_paths.go:1165]: should we mark this endpoint as deprecated?

File: vault/counters_test.go [hunk 2]
```diff
@@ -23,14 +24,22 @@ func testCountActiveTokens(t *testing.T, c *Core, root string) int {
 	t.Helper()
 
 	rootCtx := namespace.RootContext(nil)
-	resp, err := c.HandleRequest(rootCtx, &logical.Request{
+	req := &logical.Request{
```
Comment (AnPucel) [inline vault/counters_test.go:27]: Potentially newbie question. what is the advantage of this syntax over the old? Potentially just more explicit?

Comment (dhuckins) [reply vault/counters_test.go:27]: needed to use the request var further down to verify the response

Comment (AnPucel) [reply vault/counters_test.go:27]: ahhh gotcha.

Comment (dhuckins) [reply vault/counters_test.go:27]: no problem, extremely valid reason

File: vault/logical_system_paths.go (hunk 1 above)
Comment (averche) [reply vault/logical_system_paths.go:1797]: > should we mark this endpoint as deprecated?

Yes, I think so. I believe there is a `Deprecated` field at the path level.

**Review by averche — APPROVED** [review]
LGTM 👍

File: vault/logical_system_paths.go (hunk 1 above)
Comment (dhuckins) [reply vault/logical_system_paths.go:1797]: https://github.com/hashicorp/vault/pull/19359

**Pull request merged**